frontend/src/components/AvailabilityViewer/AvailabilityViewer.tsx

```diff
@@ -101,7 +101,18 @@ const AvailabilityViewer = ({ times, people, table }: AvailabilityViewerProps) =
                 people: peopleHere,
               })
             }}
+            onMouseDown={e => {
+              setTooltip({
+                anchor: e.currentTarget,
+                available: `${peopleHere.length} / ${filteredPeople.length} ${t('available')}`,
+                date: cell.label,
+                people: peopleHere,
+              })
+              let text2Clipboard = t("copy.message") + cell.label + ":\n" + peopleHere.join(';\n') + ";";
```

Comment:
After using a template for the date, you can use it here like so:

```suggestion
              const clipboardMessage = `${t('group.clipboard_message', { date: cell.label })}:\n${peopleHere.join(', ')}`
```